litmus-portal/frontend/src/models/graphql/workflowData.ts

```diff
@@ -1,10 +1,25 @@
+export interface ChaosData {
+  engineName: string;
+  engineUID: string;
+  experimentName: string;
+  experimentPod: string;
+  experimentStatus: string;
+  experimentVerdict: string;
+  failStep: string;
+  lastUpdatedAt: string;
+  namespace: string;
+  probeSuccessPercentage: string;
+  runnerPod: string;
+}
+
 export interface Node {
   children: string[] | null;
   finishedAt: string;
   name: string;
   phase: string;
   startedAt: string;
   type: string;
+  chaosData?: ChaosData;
```

Comment:
why is this an optional field?
this data is coming from network...this should always be their

Reply:
No @arkajyotiMukherjee its only available for one node... with chaos run step..